app/models/student_loans/eligibility.rb

```diff
@@ -51,6 +51,7 @@ class Eligibility < ApplicationRecord
     validates :mostly_performed_leadership_duties, on: [:"mostly-performed-leadership-duties", :submit], inclusion: {in: [true, false], message: "Select yes if you spent more than half your working hours on leadership duties"}, if: :had_leadership_position?
     validates :student_loan_repayment_amount, on: [:"student-loan-amount", :submit], presence: {message: "Enter your student loan repayment amount"}
     validates_numericality_of :student_loan_repayment_amount, message: "Enter a valid monetary amount", allow_nil: true, greater_than: 0, less_than_or_equal_to: 99999
```

Comment:
Do we need something a bit more explicit for when a SO tries to enter a value of £0, or is this too edge-casey. Now that I have written it I'm probably a 2/10 on this. 🤷‍♀️

Reply:
A value of £0 will be caught by the validation rule above this one